Code_Exercises/Exercise_16_Coalesced_Global_Memory/README.md

```diff
@@ -17,8 +17,8 @@ global memory access patterns in your kernel are coalesced.
 Consider two alternative ways to linearize the global id:
 
 ```
-auto rowMajorLinearId    = (idx[1] * width) + idx[0];  // row-major
-auto columnMajorLinearId = (idx[0] * height) + idx[1];  // column-major
+auto rowMajorLinearId    = sycl::id(globalId[0], globalId[1]);
```

Comment:
I don't think this is the original intent, here, since it talks about the linear id presumably the intent is to linearise it manually (to demonstrate the difference in striding). Can we change the other side instead to better match this?